Makefile

```diff
@@ -11,11 +10,20 @@ fq:
 .PHONY: test
 test: testgo testjq testcli
 
+.PHONY: test-race
+test-race: testgo-race testjq testcli
+
 .PHONY: testgo
 # figure out all go pakges with test files
 testgo: PKGS=$(shell find . -name "*_test.go" | xargs -n 1 dirname | sort | uniq)
 testgo:
-	go test -timeout 20m ${GO_TEST_RACE_FLAGS} ${VERBOSE} ${COVER} ${PKGS}
+	go test -timeout 20m ${VERBOSE} ${COVER} ${PKGS}
+
+.PHONY: testgo-race
+# figure out all go pakges with test files
+testgo-race: PKGS=$(shell find . -name "*_test.go" | xargs -n 1 dirname | sort | uniq)
+testgo-race:
+	go test -timeout 20m -race ${VERBOSE} ${COVER} ${PKGS}
```

Comment:
Wonder if the `testgo-race` target can be something like:
```
testgo-race: RACE=-race
testgo-race: testgo
```
Add add a `${RACE}` to `testgo`? would be nice to not repeat PKGS=.. and the `go test` command.